docs/reference/index.md

```diff
@@ -37,6 +37,7 @@ mdwrap is a command line tool to format our Markdown documentation made with
 | ---------------------------------------------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------- |
 | [ae-ano-detection](./services/ae-ano-detection.md)                     | <https://github.com/swiss-ai-center/ae-ano-detection-service>           | <https://ae-ano-detection-swiss-ai-center.kube.isc.heia-fr.ch>           | <https://ae-ano-detection-service.swiss-ai-center.ch>         |
 | [average-shade](./services/average-shade.md)                           | <https://github.com/swiss-ai-center/average-shade-service>              | <https://average-shade-swiss-ai-center.kube.isc.heia-fr.ch>              | <https://average-shade-service.swiss-ai-center.ch>            |
+| [class-rebalancing](./services/class-rebalancing.md)     | <https://github.com/swiss-ai-center/class-rebalancing-service>   | <https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch>   | <https://class-rebalancing-service.swiss-ai-center.ch> |
```

Comment:
Nitpicking here, but can we have proper spacing to get a nicer alignment? That might ease future review of that table.

Reply:
Nvm, I fixed it and cleaned the rest of the table too.